nibabel/freesurfer/mghformat.py

```diff
@@ -276,10 +276,12 @@ def set_zooms(self, zooms):
         ndims = self._ndims()
         if len(zooms) > ndims:
             raise HeaderDataError('Expecting %d zoom values' % ndims)
-        if np.any(zooms <= 0):
-            raise HeaderDataError('zooms must be positive')
+        if np.any(zooms[:3] <= 0):
+            raise HeaderDataError('zooms[:3] must be positive')
```

Comment:
I would rephrase this "Spatial zooms" instead of "zooms[:3]". Or possibly "Spatial (first three) zooms", if the position is worth noting.

Actually, it might be a good time to make the stack trace more informative, something like:

```Python
raise HeaderDataError("Spatial zooms must be positive; received {!r}".format(tuple(zooms[:3])))
```